sonar-scm-git-plugin/pom.xml

```diff
@@ -20,7 +20,7 @@
   <properties>
     <!-- Minimal supported version of SonarQube. It is defined in travis.sh 
       for integration testing too. -->
-    <sonar.apiVersion>7.0</sonar.apiVersion>
+    <sonar.apiVersion>7.4-alpha1</sonar.apiVersion>
```

Comment:
Why not 7.3? 7.4-alpha1 is not on central, so this is only if you have to use a new API.

Reply:
Yes, the new feature depends on some 7.4 API, for coverage measures on pull requests.